tests/datasets/test_deepglobelandcover.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import shutil
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pytest
+import torch
+import torch.nn as nn
+from _pytest.fixtures import SubRequest
+from _pytest.monkeypatch import MonkeyPatch
+
+from torchgeo.datasets import DeepGlobeLandCover
+
+
+class TestDeepGlobeLandCover:
+    @pytest.fixture(params=["train", "test"])
+    def dataset(
+        self, monkeypatch: MonkeyPatch, request: SubRequest
+    ) -> DeepGlobeLandCover:
+        md5 = "2cbd68d36b1485f09f32d874dde7c5c5"
+        splits = ["train", "test"]
+        monkeypatch.setattr(DeepGlobeLandCover, "md5", md5)
+        monkeypatch.setattr(DeepGlobeLandCover, "splits", splits)
```

Comment:
I don't think it's necessary to monkeypatch `splits` right? These are the same values as the default